test-owasp.sh

```diff
@@ -1,10 +1,29 @@
 #!/bin/bash
 
-zap-cli start --start-options '-config api.disablekey=true'
-zap-cli open-url $LIVE_TARGET
-zap-cli spider $LIVE_TARGET
-zap-cli active-scan -s all -r $LIVE_TARGET
-zap-cli alerts -l Informational -f json > results/zap.json
-zap-cli shutdown
-
-exit 0
+apt-get install jq
+pip install tinys3
+
+COUNTER=0
+COUNT=$(cat scripts/targets.json | jq '.targets[] .url' | wc -l)
+
+
+while [ $COUNTER -lt $COUNT ]; do
+  NAME=$(cat scripts/targets.json | jq ".targets[${COUNTER}] .name" -r)
+  TARGET=$(cat scripts/targets.json | jq ".targets[${COUNTER}] .url" -r)
+
+  echo Scanning $NAME: $TARGET
+  # Leaving ZAP up between runs allows alerts to accrue
+  zap-cli start --start-options '-config api.disablekey=true'
+  zap-cli open-url $TARGET
+  zap-cli spider $TARGET
+  zap-cli active-scan -s all -r $TARGET
+  zap-cli alerts -l Informational -f json > results/${NAME}.json
+  zap-cli shutdown
+
+  let COUNTER+=1
+done
```

Comment:
I think this loop can be simplified to:

```bash
zap-cli start --start-options '-config api.disablekey=true'

while [ $COUNTER -lt $COUNT ]; do
  # ...

  zap-cli quick-scan --spider --scanners all $TARGET
  zap-cli alerts -l Informational -f json > results/${NAME}.json
  zap-cli session new

  # ...
done
```

which will also make the task faster, since it isn't restarting ZAP for each iteration. Not sure if it matters if the `quick-scan` vs. `active-scan` command is used...?